### PR TITLE
Release 1.0.0.9

### PR DESCRIPTION
**Pull request by quoctruong**
This change is [Reviewable](https://reviewable.io/reviews/googlecloudplatform/google-cloud-powershell/555)

### CODE REVIEW

File: Google.PowerShell/ReleaseFiles/GoogleCloud.psd1 [hunk 1]
```diff
@@ -3,7 +3,7 @@
 #
 # Generated by: Google Inc
 #
-# Generated on: 6/15/2017
+# Generated on: 8/7/2017
```
Comment (iantalarico) [inline Google.PowerShell/ReleaseFiles/GoogleCloud.psd1]: Should a version number be updated?

File: Google.PowerShell/ReleaseFiles/GoogleCloud.psd1 [hunk 2]
```diff
@@ -66,7 +66,7 @@ FormatsToProcess = 'GoogleCloudPlatform.Format.ps1xml'
 # NestedModules = @()
 
 # Functions to export from this module
-FunctionsToExport = @('gs:')
+FunctionsToExport = 'gs:'
```
Comment (iantalarico) [inline Google.PowerShell/ReleaseFiles/GoogleCloud.psd1]: Why this change?

File: Tools/BuildAndPackage.ps1 [hunk 3]
```diff
@@ -78,7 +78,7 @@ function CleanUp-PrivateDataAndExportVariables($moduleManifest) {
 
     $moduleManifest.Remove("PrivateData")
     $moduleManifest["VariablesToExport"] = $toBeReplaced
-    $moduleManifest["FunctionsToExport"] = $toBeReplaced
+    $moduleManifest["FunctionsToExport"] = @("gs:")
```
Comment (iantalarico) [inline Tools/BuildAndPackage.ps1]: Why this change?  Should this reference FunctionsToExport ?

**Comment by quoctruong** [comment]
@iantalarico PTAL.

File: Tools/BuildAndPackage.ps1 (hunk 3 above)
Comment (quoctruong) [inline Tools/BuildAndPackage.ps1]: I removed it now. It should just read it from the file. Previously, we want it blank.

File: Google.PowerShell/ReleaseFiles/GoogleCloud.psd1 (hunk 2 above)
Comment (quoctruong) [inline Google.PowerShell/ReleaseFiles/GoogleCloud.psd1]: This is auto-generated by a script (the one before was hand-written).

File: Google.PowerShell/ReleaseFiles/GoogleCloud.psd1 (hunk 1 above)
Comment (quoctruong) [inline Google.PowerShell/ReleaseFiles/GoogleCloud.psd1]: Done. I changed it to the wrong version so that's why it didn't get updated.

**Comment by quoctruong** [comment]
Thanks!
